demo/src/main/java/com/bluelinelabs/conductor/demo/util/AnimUtils.java

```diff
@@ -47,25 +47,28 @@ private AnimUtils() { }
     private static Interpolator linear;
 
     @NonNull
-    public static Interpolator getFastOutSlowInInterpolator() {
+    public static Interpolator getFastOutSlowInInterpolator(Context context) {
```

Comment:
You can def re-revert this. I was playing around with the idea that doing it exactly the way Plaid did it somehow mattered. It doesn't, but I never changed it back.